_includes/footer.html

```diff
@@ -39,8 +39,19 @@
     </div>
   </div>
   <div class="bottom">
-    <a class="logo" href="{{ "/" | relative_url }}"><img src="{{ "/images/rook-logo-small.svg" | relative_url }}" alt="rook.io" /></a>
-    <span>&#169; {{ site.time | date: '%Y' }} Rook Authors. All rights reserved.</span>
+    <div class="grid-center">
+      <div class="col-6">
+        <a class="logo" href="{{ "/" | relative_url }}"><img src="{{ "/images/rook-logo-small.svg" | relative_url }}" alt="rook.io" /></a>
+
+        <br />
+
+        <span>&#169; Rook Authors {{ site.time | date: '%Y' }}. Documentation distributed under <a href="https://creativecommons.org/licenses/by/4.0">CC-BY-4.0</a>.</span>
```

Comment:
Is it typical for other CNCF/LF projects to do docs under creative commons but code under apache 2?

Reply:
This is the standard language, as determined by LF legal. You can see it in other CNCF projects like [Prometheus](https://prometheus.io), [Kubernetes](https://kubernetes.io), etc.